tests/test_ctx.py

```diff
@@ -361,9 +371,42 @@ def test_add_basemap():
     )
     assert ax.get_xlim() == (x1, x2)
     assert ax.get_ylim() == (y1, y2)
-    assert ax.images[0].get_array().sum() == 464751694
-    assert ax.images[0].get_array().shape == (980, 862, 3)
-    assert_array_almost_equal(ax.images[0].get_array().mean(), 183.38608756727749)
+
+    assert ax.images[0].get_array()[:,:,:3].sum() == 464536503
+    assert ax.images[0].get_array().shape == (980, 862, 4)
+    assert_array_almost_equal(ax.images[0].get_array()[:,:,:3].mean(), 183.301175)
+
+	#<<<<<<< HEAD
+    assert ax.images[0].get_array().sum() == 678981558
+    assert_array_almost_equal(ax.images[0].get_array().mean(), 200.939189)
+
+    x1, x2, y1, y2 = [
+        -11740727.544603072,
+        -11701591.786121061,
+        4852834.0517692715,
+        4891969.810251278,
+    ]
+
+    # Test web basemap with overlay layer
```

Comment:
Can you start a new test function for this? (eg `def test_add_basemap_overlay`, the existing one is already a bit too long)

Reply:
Test is broken out. Also fixed the test so it points to Boulder (where there are labels) and not the Atlantic Ocean (where there's nothing).